include/hwinfo/cpu.h

```diff
@@ -36,6 +36,7 @@ class CPU {
   ~CPU() = default;
 
   int id() const;
+  const std::string& processorId() const;
```

Comment:
Not sure if this information is useful at all. Maybe just remove it for the time being